services/wallet/collectibles/collectible_data_db.go

```diff
@@ -0,0 +1,164 @@
+package collectibles
+
+import (
+	"database/sql"
+	"fmt"
+	"math/big"
+
+	"github.com/status-im/status-go/services/wallet/bigint"
+	"github.com/status-im/status-go/services/wallet/thirdparty"
+)
+
+type CollectibleDataDB struct {
+	db *sql.DB
+}
+
+func NewCollectibleDataDB(sqlDb *sql.DB) *CollectibleDataDB {
+	return &CollectibleDataDB{
+		db: sqlDb,
+	}
+}
+
+const collectibleDataColumns = "chain_id, contract_address, token_id, provider, name, description, permalink, image_url, animation_url, animation_media_type, traits, background_color, token_uri"
+
+func insertOrReplaceCollectiblesData(creator statementCreator, collectibles []thirdparty.CollectibleData) error {
+	insertCollectible, err := creator.Prepare(fmt.Sprintf(`INSERT OR REPLACE INTO collectible_data_cache (%s) 
+																				VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`, collectibleDataColumns))
```

Comment:
is it me or indent is odd?

Reply:
hmm the format is auto-set every time I save, there's a tab in there